Examples/Python/ellipsoid.py

```diff
@@ -240,6 +240,20 @@ def Run_Pipeline(args):
         dt_files = sw.utils.save_images(groom_dir + 'distance_transforms/', shape_seg_list,
                                         shape_names, extension='nrrd', compressed=False, verbose=True)
 
+    # If running optimzation in mesh mode, convert distance transforms to meshes
```

Comment:
Please create a helper function for this and the identical instances of it elsewhere in this PR.

Reply:
I thought about doing this... Ultimately this is kind of part of the grooming process which we decided to make transparent when we switched to pybind and got rid of groom utils. I could make a helper function in the python package but this adds a layer of abstraction and makes the use cases less readable... what do you think?

Reply:
I think that it's complex enough that it warrants a single function.